plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts

```diff
@@ -44,6 +45,7 @@ type NunjucksWorkflowRunnerOptions = {
   actionRegistry: TemplateActionRegistry;
   integrations: ScmIntegrations;
   logger: winston.Logger;
+  additionalFilters?: Record<string, NunjucksFilter>;
```

Comment:
Probably makes sense to call it `nunjucksFilters` here as well

Reply:
I was a bit iffy on that one. It felt somewhat redundant to call it a nunjucks filter in the `NunjucksWorkflowRunner`, but I'm fine either way. Change is applied and pushed.